base/ltfssaxes.dtx

```diff
@@ -1175,14 +1177,25 @@
 %  \begin{macro}{\sscshape}
 %  \begin{macro}{\textssc}
 %  \begin{macro}{\sscdefault}
+%    
 %    New command to select spaced small capitals. This is only here
 %    because \texttt{fontaxes} offered it. There isn't a single free
 %    font that supports it. However, some commercial ones do, so we
-%    offer it so that at some point \texttt{fontaxes} could be
-%    retired.
+%    offer it and it is possible to produce spaced small capitals from
+%    normal small capitals in OTF fonts using \texttt{otftotfm} with
+%    calls such as
+%\begin{verbatim}
+%  otftotfm -e TEXMF/fonts/enc/dvips/base/texnansx.enc \
+%    SourceSerifPro-Regular.ttf -fkern -fliga --feature=smcp \
+%    --letterspacing=80 SourceSerifPro-Regular-ssc-LY1
+%\end{verbatim}
 %
-%    So far there aren't any rules for it---probably there should be
-%    some putting it in the same category as \texttt{sc}.
+%    Michael Ummels kindly prepared the necessary rules for
+%    \texttt{ssc} ages ago, but until recently they managed to hide
+%    deep down in my inbox. I
+%    finally got around integrating them (with a few changes) and also
+%    took the opportunity to rationalize the other uncommon
+%    \texttt{sw} shape rules a bit.
 %    \begin{macrocode}
```

Comment:
Rearrangement:
. . . to rationalize (a bit) the rules for the only other uncommon shape, \texttt{sw}.